tests/wakunode2/test_app.nim

```diff
@@ -53,3 +55,28 @@ suite "Wakunode2 - App initialization":
 
     ## Then
     check res.isOk()
+
+  test "node setup is successful with default configuration":
```

Comment:
I cannot use `asyncTest` here due to some non-GC-safe issue that makes the `wakunode2.setupWakuArchive()` incompatible with it. 

I am using `waitFor` as a temporary workaround.